packages/thirdweb/src/indexer/endpoints/getLatestBlock.ts

```diff
@@ -0,0 +1,6 @@
+// TODO clarify the types, no need to sanitize here because we are already doing it server-side?
+export async function getLatestBlock(client: any, params: any) {
```

Comment:
I think this should be `getLatestBlocknumber`?

Reply:
Changed in [implement getLatestBlockNumber for chainsaw](https://github.com/thirdweb-dev/js/pull/3741/commits/22d0c2c627c21234af979250b4ba62262617d507)